pkg/queue/cluster_queue_interface.go

```diff
@@ -43,6 +43,10 @@ type ClusterQueue interface {
 	// PushOrUpdate pushes the workload to ClusterQueue.
 	// If the workload is already present, updates with the new one.
 	PushOrUpdate(*kueue.QueuedWorkload)
+	// PushIfNotPresent pushes the workload to ClusterQueue.
+	// If the workload is already present, returns false. Otherwise returns true.
+	PushIfNotPresent(*workload.Info) bool
```

Comment:
Instead of adding this function, rename `AddInadmissibleIfNotPresent` to `RequeueIfNotPresent(workload *workload.Info, inadmissible bool)`

Reply:
Updated.